src/python/turicreate/data_structures/image.py

```diff
@@ -226,7 +226,7 @@ def save(self, filename):
 
     def show(self):
         """
-        Displays the image. Requires PIL/Pillow.
```

Comment:
Why remove this comment? I think it is still true.

Reply:
It is still true, but PIL/Pillow is now a hard dependency. So I don't think it makes sense to mention.

Reply:
I see. Makes sense, thanks!